cfme/tests/infrastructure/test_vm_clone.py

```diff
@@ -0,0 +1,55 @@
+# -*- coding: utf-8 -*-
+import pytest
+
+from utils.providers import setup_a_provider as _setup_a_provider
+from cfme.infrastructure.virtual_machines import Vm
+from cfme.services import requests
+from utils.wait import wait_for
+from utils import testgen
+from utils.log import logger
+from utils.randomness import generate_random_string
+
+pytestmark = [
+    pytest.mark.fixtureconf(server_roles="+automate")
+]
+
+pytest_generate_tests = testgen.generate(testgen.provider_by_type, 'virtualcenter', scope="module")
+
+
+@pytest.fixture(scope="module")
+def setup_a_provider():
+    _setup_a_provider("infra")
```

Comment:
you should be able to use module fixture,setup_provider, from fixtures/provider.py